pkg/plugins/steady/steady.go

```diff
@@ -0,0 +1,68 @@
+package steady
+
+import (
+	"context"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/selection"
+	"k8s.io/apimachinery/pkg/util/sets"
+	clusterapiv1 "open-cluster-management.io/api/cluster/v1"
+	clusterapiv1alpha1 "open-cluster-management.io/api/cluster/v1alpha1"
+	"open-cluster-management.io/placement/pkg/plugins"
+)
+
+const placementLabel = "cluster.open-cluster-management.io/placement"
+
+var _ plugins.Prioritizer = &Steady{}
+
+type Steady struct {
+	meta   metav1.ObjectMeta
+	handle plugins.Handle
+}
+
+func New(handle plugins.Handle) *Steady {
+	return &Steady{handle: handle}
+}
+
+func (s *Steady) Name() string {
+	return "steady"
+}
+
+func (s *Steady) Set(pl *clusterapiv1alpha1.Placement) {
+	s.meta = pl.ObjectMeta
+}
+
+func (s *Steady) Score(ctx context.Context, clusters []*clusterapiv1.ManagedCluster) (map[string]float64, error) {
+	// query placementdecisions with label selector
+	scores := map[string]float64{}
+	requirement, err := labels.NewRequirement(placementLabel, selection.Equals, []string{s.meta.Name})
```

Comment:
I'm not understanding how this s.meta is ensured to be set prior to calling Score.  

I would not suggest passing stateful information to a single instance and counting on call order.